src/points/PointsHistoryBottomSheet.tsx

```diff
@@ -165,11 +183,27 @@ function PointsHistoryBottomSheet({ forwardedRef }: Props) {
         ListEmptyComponent={renderEmpty ? EmptyOrError : null}
         onEndReachedThreshold={0.5}
       />
+      {showErrorBanner && (
+        <InLineNotification
+          variant={NotificationVariant.Error}
+          title={t('points.history.pageError.title')}
+          description={t('points.history.pageError.subtitle')}
+          ctaLabel={t('points.history.pageError.refresh')}
+          onPressCta={onPressRefresh}
+          withBorder={true}
+          style={styles.errorBanner}
+          customIcon={<AttentionIcon color={colors.errorDark} size={20} />}
+          testID={'PointsHistoryBottomSheet/ErrorBanner'}
+        />
+      )}
     </BottomSheetBase>
   )
 }
 
 const styles = StyleSheet.create({
+  errorBanner: {
+    marginHorizontal: Spacing.Regular16,
+  },
```

Comment:
I suggest we use absolute positioning (over the content) toward the bottom for this one.
So it doesn't abruptly push the content up when it appears.